Formula/gdk-pixbuf.rb

```diff
@@ -13,6 +13,10 @@ class GdkPixbuf < Formula
 
   option :universal
   option "with-relocations", "Build with relocation support for bundles"
+  option "without-modules", "Disable dynamic module loading"
+  option "with-included-loaders=", "Build the specified loaders into gdk-pixbuf"
+
+  deprecated_option "disable-modules" => "without-modules"
```

Comment:
`deprecated_option` is for options that existed at some point, which is not the case here.